app/Http/Controllers/StateController.php

```diff
@@ -4,7 +4,23 @@
 
 use Illuminate\Http\Request;
 
+use App\Model\State;
+use App\Model\Country;
+
 class StateController extends Controller
 {
-    //
-}
+    public function getStates($country){
+    	$getCountry = Country::where('name', $country)->first();
+    	if ($getCountry) {
+    		return response()->json([
+	            'data'=> State::where('country_id', $getCountry->id)->get(),
+	            'message' => 'All countries gotten Succesfully'
```

Comment:
Are you getting states here or country, look at the message in your return statement